examples/England-PractitionerRole-Pass-Example.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PractitionerRole xmlns="http://hl7.org/fhir">
+  <id value="England-PractitionerRole-Pass-Example" />
```

Comment:
Again, GeneralPractitioner, this example is almost same thing with England-PractitionerRole-NHS-Example. This one is more detailed and can be retained as England-PractitionerRole-GeneralPractitioner-Example

Reply:
Pushed change. Renamed to be more appropriate